Sources/TripKitUI/views/trip overview/TKUINotificationView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  TKUINotificationView.swift
+//  TripKitUI-iOS
+//
+//  Created by Jules Ian Gilos on 12/1/22.
+//  Copyright © 2022 SkedGo Pty Ltd. All rights reserved.
+//
+
+import UIKit
+
+import TripKit
+import RxSwift
+
+class TKUINotificationView: UIView {
+  
+  @IBOutlet private weak var contentWrapper: UIView!
+
+  @IBOutlet weak var notificationSwitch: UISwitch!
+  
+  @IBOutlet weak var titleImageView: UIImageView!
+  @IBOutlet weak var titleLabel: UILabel!
+  
+  @IBOutlet weak var detailTitleLabel: UILabel!
+  
+  @IBOutlet weak var detailView: UIView!
+  
+  // Assuming this is constant first
+  @IBOutlet weak var detailItem1: UILabel!
+  @IBOutlet weak var detailItem2: UILabel!
+  @IBOutlet weak var detailItem3: UILabel!
+  @IBOutlet weak var detailItem4: UILabel!
```

Comment:
This should also depend on what's in `trip.segments.flatMap(\.geofences)`, in particular which types.

Reply:
I did that in https://github.com/skedgo/tripkit-ios/pull/232 by fading out those items which aren't supported for the trip.

Reply:
Oh okay, will relook, I thought this was only a xib change.